### DISCUSSION TITLE
Expedition 2 is not showing any rewards

### DISCUSSION THREAD

**Discussion by womenizer97** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#44] [Q&A] [answered]
<img width="1440" alt="Screenshot 2024-11-14 at 8 13 16 PM" src="https://github.com/user-attachments/assets/4f5ccc90-ee6e-420f-aa16-5f46d42e737f">

**Comment by womenizer97** [comment]
I tried testing as well, completed phase 1 and 2, still couldnt see the rewards in the main save

**Reply by cwmonkey** [reply]
@womenizer97 Hm, I don't even know how to test this since I'd have to remove the rewards from my account first. Do you see the rewards when you remove the filter? (Just look at all of them and scroll down)

**Reply by womenizer97** [reply]
Using your file/json files on the website you created, can we do the expedition twice? Coz I see my phase 1 reward available on primary save dressing modifier but not in the quicksilver vendor list...it only shows the screenshot I added...also couldn't see any of those rewards even with or without the filters

**Reply by cwmonkey** [reply, marked as answer]
I just tested and I can see my Beachhead rewards under the "Expedition 2" filter, HOWEVER I did complete the new expedition 2 (while offline) after first downloading the new .json file from the HG servers. It could be if you use the old expedition 2 json file the rewards don't properly register? Like I said it would be hard to find out.

To answer your question, you should be able to do the same expedition multiple times on the same save. I actually had to do that to one of my saves because I was on the old exp 2 when the new exp 2 started, and things got real weird.

To run an expedition again do the following:

1. Go to https://cwmonkey.github.io/nms-expeditions/ and download an expedition you DON'T want to run, for example expedition 1
2. Put the expedition 1 SEASON_DATA_CACHE.JSON in your cache folder
3. While offline, open the game, load your main save, go to the Anomaly and start expedition 1
4. Get to your ship and get in/out so your game saves, then switch back to your main save in the pause menu
5. Get in/out of your ship in the Anomaly on your main save and close the game
6. Download the expedition you DO want to run (expedition 2)
7. Put the expedition 2 SEASON_DATA_CACHE.JSON in your cache folder
8. While offline, open the game, load your main save. You should now be able to start expedition 2 from the terminal again

There may be an easier way to do this with save editing, but I haven't tried.

**Reply by womenizer97** [reply]
I believe the online one, nothing has changed from the previous ones...if there are changes....would you please update the json on the website....

Also if I complete the expedition twice, what happens to the rewards


Just want the Normandy that's all

**Reply by cwmonkey** [reply]
@womenizer97 A LOT has changed from the first one. In fact, the first one doesn't really work anymore since they changed the Normandy location to be closer to the new starting planet.

You can see the diff here in the `02_redux_beachhead.json` file: https://github.com/cwmonkey/nms-expeditions/commit/0440d57108e820b545c68c32873b9d11ad38a99d

It has 2,391 new lines and 506 deleted lines.

Anyway, it was added to https://cwmonkey.github.io/nms-expeditions/ and this repository six days ago, so the above instructions should work for you.

**Reply by womenizer97** [reply]
I just downloaded and started today, so I have the updated one, thanks

**Comment by womenizer97** [comment]
Just completed the expedition and now can see all the rewards. Thanks man
